src/useKeyboard.ts

```diff
@@ -15,7 +15,7 @@ const initialValue = {
 export function useKeyboard() {
   const [shown, setShown] = useState(false)
   const [coordinates, setCoordinates] = useState<{
-    start: ScreenRect
+    start: undefined | ScreenRect
```

Comment:
It seems like it is, although this only brings the types in line with reality.

I had to update the react-native types because `getSizeWithHeaders` didn't exist in the older version of the types yet, but the update also pulled in a correction in the types of `KeyboardEvent`, which is fixed with this `undefined`.

Long story short: yes it is a breaking change, but it'll only help people who were now relying on start always being a `ScreenRect`, which isn't the case on android.